vpr/src/route/rr_graph_generation/rr_graph_opin_chan_edges.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <vector>
+#include "rr_types.h"
+#include "rr_graph_type.h"
+#include "rr_edge.h"
+
+class RRGraphBuilder;
```

Comment:
Add Doxygen comment for what should go in this file (if done here, don't need to do in the .cpp file)